scripts/prestart.js

```diff
@@ -0,0 +1,21 @@
+const axios = require('axios');
```

Comment:
something seems not to work with this.
I think its not getting built when using typescript, because its not in the `src`-Folder.
I dont exactly know what the linter doesnt like about that.

Reply:
can this folder be ignored?

```diff
diff --git a/.eslintrc.js b/.eslintrc.js
index 64d1185..7092452 100644
--- a/.eslintrc.js
+++ b/.eslintrc.js
@@ -15,7 +15,7 @@ module.exports = {
     node: true,
     jest: true,
   },
-  ignorePatterns: ['.eslintrc.js'],
+  ignorePatterns: ['.eslintrc.js', 'scripts/*.js'],
   rules: {
     '@typescript-eslint/interface-name-prefix': 'off',
     '@typescript-eslint/explicit-function-return-type': 'off',
diff --git a/tsconfig.json b/tsconfig.json
index 3249a47..83c2551 100644
--- a/tsconfig.json
+++ b/tsconfig.json
@@ -12,5 +12,5 @@
     "incremental": true,
     "allowSyntheticDefaultImports": true
   },
-  "exclude": ["node_modules", "dist"]
+  "exclude": ["node_modules", "dist", "scripts"]
 }
```